service/src/main/java/bio/terra/workspace/app/controller/ControlledAzureResourceApiController.java

```diff
@@ -5,26 +5,14 @@
 import bio.terra.workspace.common.utils.ControllerUtils;
 import bio.terra.workspace.db.exception.InvalidMetadataException;
 import bio.terra.workspace.generated.controller.ControlledAzureResourceApi;
-import bio.terra.workspace.generated.model.ApiAccessScope;
-import bio.terra.workspace.generated.model.ApiAzureIpResource;
-import bio.terra.workspace.generated.model.ApiControlledResourceCommonFields;
-import bio.terra.workspace.generated.model.ApiCreateControlledAzureIpRequestBody;
-import bio.terra.workspace.generated.model.ApiCreatedControlledAzureIp;
-import bio.terra.workspace.generated.model.ApiDeleteControlledAzureIpRequest;
-import bio.terra.workspace.generated.model.ApiDeleteControlledAzureIpResult;
-import bio.terra.workspace.generated.model.ApiJobControl;
-import bio.terra.workspace.generated.model.ApiPrivateResourceUser;
+import bio.terra.workspace.generated.model.*;
```

Comment:
I forgot to mention, `*` imports are not allowed per code style. You can change the IDE to not generate them by setting these 2 fields to 999:

![image](https://user-images.githubusercontent.com/5368863/137920466-9a9447ee-0db1-43e6-9d8a-9c81090a7e50.png)